docs/install-audit-log-filter.md

```diff
@@ -0,0 +1,43 @@
+# Install the Audit Log Filter
+
+The feature is in [tech preview](glossary.md#tech-preview).
+
+The `plugin_dir` system variable defines the plugin library location. If needed, at server startup, set the `plugin_dir` variable.
+
+When upgrading a MySQL installation, plugins are not automatically upgraded. You may need to manually load the plugin after the MySQL upgrade.
+
+In the `share` directory, locate the `audit_log_filter_linux_install.sql `script.
+
+Implemented in 8.0.34, at the time you run the script, you can select the database used to store the JSON filter tables. 
+
+The `-D` option in the installation script defaults to `mysql` except for the following scenarios:
```

Comment:
Not sure it is correct to say "`-D` option in the installation script defaults to `mysql`". This is the mysql client's option actually. It may be better to remove this line and add the third scenario below:

* If the plugin is not loaded and `-D` option is not provided, the database name where installation script creates required tables defaults to 'mysql'.